src/main/java/reposense/RepoSense.java

```diff
@@ -82,6 +83,8 @@ public static void main(String[] args) {
             logger.log(Level.WARNING, ioe.getMessage(), ioe);
         } catch (ParseException pe) {
             logger.log(Level.WARNING, pe.getMessage(), pe);
+        } catch (InvalidCsvException ice) {
```

Comment:
@fzdy1914, should we combine the catch clauses for 
`IOException`, `ParseException`, `InvalidCsvException` ? 
The logic to handle these exceptions is the same.

Reply:
Yes, please.

https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/parser/InvalidCsvException.java#L6

I am also wondering if we should change the `InvalidCsvException` to extend `IOException` instead of generic `Exception`

Reply:
> I am also wondering if we should change the InvalidCsvException to extend IOException instead of generic Exception


I am not sure if this is suitable. `IOExceptions` are thrown for errors that prevent us from accessing / writing a file. The `InvalidCsvException` is thrown when we can access the csv file, but the csv's contents are not in line with RepoSense's expectations.

Reply:
@fzdy1914, I have collapsed the catch blocks